Update and rename `osu!dev Discord server` to `osu! Discord server` (#9176)
ppy/osu-wiki: wiki/Community/osu!_Discord_server/en.md

---
tags:
  - development
  - osu!dev
---

> # osu Discord server

Reviewer: why osu without exclamation?
```suggestion
# osu! Discord server
```

Author: my fault lol

> # osu Discord server

Reviewer 2: ```suggestion
# osu! Discord server
```

> The **osu! Discord server**, previously known as **osu!dev**, is osu!'s official [Discord](https://discord.com/) server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the [osu! team](/wiki/People/osu!_team).

Reviewer 2: ```suggestion
The **osu!** Discord server, previously known as **osu!dev**, is osu!'s official [Discord](https://discord.com/) server for development-related purposes. The server functions as a hub for people to discuss and work on osu!'s open source and community projects. This is the main place to contribute to osu!'s development and get in touch with the [osu! team](/wiki/People/osu!_team).
```

The permanent invite link for the server is <https://discord.gg/ppy>.

People new to the server are encouraged to introduce themselves in the `#introductions` channel, but doing so is not a requirement to interact with the rest of the server.

## Channels

> The osu!dev Discord server features a number of channels, many of which are centered around the development of a specific project or two.

Reviewer 2: ```suggestion
The osu!dev Discord server features a number of channels, many of which are centred around the development of a specific project or two.
```

> The osu!dev Discord server features a number of channels, many of which are centered around the development of a specific project or two.

Reviewer: I think you forgot to remove it from here
```suggestion
The osu! Discord server features a number of channels, many of which are centered around the development of a specific project or two.
```

### General

> | Channel | Project |

Reviewer 2: "Description" instead of "Project" on all of these

| :-- | :-- |

> | `#welcome` | *N/A* |
> | `#introduction` | *N/A* |
> | `#general` | *N/A* |

Reviewer 2: ```suggestion
| `#welcome` | Server information |
| `#introduction` | Self-introductions |
| `#general` | General discussion and troubleshooting related to osu! |
```

### Client / Web

| Channel | Project |
| :-- | :-- |
| `#osu` | [osu!(lazer)](/wiki/Client/Release_stream/Lazer) |
| `#osu-framework` | [osu!framework](https://github.com/ppy/osu-framework) |
| `#osu-web` | [osu!web](https://github.com/ppy/osu-web) |

> | `#github-feed` | *N/A* |

Reviewer 2: ```suggestion
| `#github-feed` | Event feed for various [ppy GitHub repositories](https://github.com/ppy) |
```

> | `Meetings` (Stage channel) | [osu! community meetings](/wiki/Community/osu!_community_meetings) |

Reviewer 2: those "osu! dev catch up" meetings

```suggestion
| `Meetings` (stage channel) | [osu! community meetings](/wiki/Community/osu!_community_meetings) and developer progress meetings |
```

Author: "Stage" is capitalized according to Discord (http://support.discord.com/hc/en-us/articles/1500005513722-Stage-Channels-FAQ) so I'll be keeping that as is i guess
